dashboard/scripts/archive/backfill_script_level_ids_teacher_feedbacks_4.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# Backfill existing TeacherFeedbacks to set script_level_id based on script
+# stability. Scripts have is_stable set to true when we launch curriculum.
+# Scripts that do not have is_stable set to true are used internally, usually
```

Comment:
this strategy seems fine given that this is a one-off script. however you wouldn't want to make this assumption in the month of May, when new versions of our courses are visible but not yet stable.